src/main/scala/org/broadinstitute/dsde/firecloud/webservice/CookieAuthedApiService.scala

```diff
@@ -1,38 +1,40 @@
 package org.broadinstitute.dsde.firecloud.webservice
 
+import akka.actor.Props
+import com.typesafe.scalalogging.slf4j.LazyLogging
 import org.broadinstitute.dsde.firecloud.dataaccess.HttpGoogleServicesDAO
 import org.broadinstitute.dsde.firecloud.model.UserInfo
 import org.broadinstitute.dsde.firecloud.service._
-import org.slf4j.LoggerFactory
-import spray.http.OAuth2BearerToken
+import spray.http._
 import spray.routing._
 
+import scala.language.postfixOps
+
 /**
  * Created by dvoet on 11/16/16.
  */
-trait CookieAuthedApiService extends HttpService with PerRequestCreator with FireCloudDirectives
-  with FireCloudRequestBuilding {
+trait CookieAuthedApiService extends HttpService with FireCloudDirectives with FireCloudRequestBuilding with LazyLogging {
 
   val exportEntitiesByTypeConstructor: UserInfo => ExportEntitiesByTypeActor
 
   private implicit val executionContext = actorRefFactory.dispatcher
-  lazy val log = LoggerFactory.getLogger(getClass)
 
-  def cookieAuthedRoutes: Route =
+  val cookieAuthedRoutes: Route =
     // download "proxy" for TSV files
-    path("workspaces" / Segment / Segment/ "entities" / Segment/ "tsv") {
+    path( "cookie-authed" / "workspaces" / Segment / Segment/ "entities" / Segment / "tsv" ) {
       (workspaceNamespace, workspaceName, entityType) =>
         formFields('FCtoken, 'attributeNames.?) { (tokenValue, attributeNamesString) =>
           post { requestContext =>
-            val filename = entityType + ".tsv"
             val attributeNames = attributeNamesString.map(_.split(",").toIndexedSeq)
             val userInfo = UserInfo("dummy", OAuth2BearerToken(tokenValue), -1, "dummy")
-            perRequest(requestContext, ExportEntitiesByTypeActor.props(exportEntitiesByTypeConstructor, userInfo),
-              ExportEntitiesByTypeActor.ExportEntities(workspaceNamespace, workspaceName, filename, entityType, attributeNames))
+            val exportProps: Props = ExportEntitiesByTypeActor.props(exportEntitiesByTypeConstructor, userInfo)
+            val exportMessage = ExportEntitiesByTypeActor.ExportEntities(requestContext, workspaceNamespace, workspaceName, entityType, attributeNames)
+            val exportActor = actorRefFactory.actorOf(exportProps)
+            exportActor ! exportMessage
```

Comment:
It wasn't being used any longer. If there's a way to wrap the streaming download into PerRequest, I'd love to see how to do it. I can't envision how that would work.